src/Bicep.LangServer/Deploy/DeploymentHelper.cs

```diff
@@ -27,8 +27,8 @@ public class DeploymentHelper
         /// <param name="id">id string to create the ResourceIdentifier from</param>
         /// <param name="scope">target scope</param>
         /// <param name="location">location to store the deployment data</param>
-        /// <returns>deployment succeeded/failed message</returns>
-        public static async Task<string> CreateDeployment(
+        /// <returns>deployment succeeded/failed message with result</returns>
+        public static async Task<(string, string)> CreateDeployment(
```

Comment:
Please use named tuplet

Reply:
That way it's self-documenting.

Reply:
I'm actually wondering though if it makes sense to instead return a true/false for result, and translate to Succeeded/Failed when posting the telemetry.   Fewer places where you could end up with an invalid value.

Reply:
... and no chance of mixing up the result string with the message string.

Reply:
Updated here:
https://github.com/Azure/bicep/commit/c15921c329b811f4103d01048c35655d57d195ea

Thanks!